Monika After Story/game/chess.rpy

```diff
@@ -179,7 +179,9 @@ init 1 python in mas_chess:
         "Tch-",
         "Winning isn't everything, you know...",
         "Ahaha,{w} I was just letting you win since you keep losing so much.",
-        "Oh, you won.{w} I should have taken this game seriously, then."
+        "Oh, you won.{w} I should have taken this game seriously, then.",
```

Comment:
First of all, I'm not sure if any of these are even still used. 

Second, I really don't think we want Monika to be this mean about a normal game of chess.